apps/document-generation/src/templates/credit_report_dispute_letter/0.pug

```diff
@@ -0,0 +1,57 @@
+-
+  var moment = UTILS.moment;
```

Comment:
Should we remove these files and add them when we are actually porting the templates? @duranmla

Reply:
Yes, I think you are right. BTW any ideas regarding the name of the template? Or makes sense to keep the 0, 1, 2...